openwisp_controller/config/tests/test_vpn.py

```diff
@@ -151,6 +151,46 @@ def test_vpn_client_cert_post_deletes_cert(self):
         self.assertEqual(VpnClient.objects.filter(pk=vpnclient.pk).count(), 0)
         self.assertEqual(Cert.objects.filter(pk=cert_pk).count(), 0)
 
+    def test_vpn_client_post_deletes_auto_cert(self):
```

Comment:
I would rename this to `test_vpn_client_deletion` and move it after `test_auto_client_auto_cert_False`.

The rest looks good!